ownCloud/Theming/ThemeCollection.swift

```diff
@@ -174,6 +175,7 @@ class ThemeCollection : NSObject {
 		self.tableBackgroundColor = UIColor.white
 		self.tableGroupBackgroundColor = UIColor.groupTableViewBackground
 		self.tableSeparatorColor = nil
+		self.tableRowBorderColor = UIColor.black.withAlphaComponent(0.1)
```

Comment:
@hosy I think it makes sense to create a constant for 0,1 since it is also used multiple times

Reply:
Ok

Reply:
Just checked the code, but for me this are individual values for every color. I cannot reuse this values for the other colors, because it I change the constant, other colors will also get a change, which may not be wanted.